src/test/java/io/appium/java_client/pagefactory_tests/widget/tests/WidgetTest.java

```diff
@@ -1,25 +1,22 @@
 package io.appium.java_client.pagefactory_tests.widget.tests;
 
-import static java.util.Arrays.copyOf;
 import static org.openqa.selenium.support.PageFactory.initElements;
 
 import io.appium.java_client.pagefactory.AppiumFieldDecorator;
-import org.junit.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 import org.openqa.selenium.WebDriver;
 
 public abstract class WidgetTest {
 
     protected final AbstractApp app;
 
-    protected static Object[] dataArray(Object...objects) {
-        return copyOf(objects, objects.length);
-    }
-
     protected WidgetTest(AbstractApp app, WebDriver driver) {
```

Comment:
I've checked, tests are not run with old approach, they require refactoring

Reply:
@valfirst  I have reverted Widget tests to run in old approach using Vintage Engine.

Reply:
Can we refactor thsese changes in another PR?

Reply:
which issues are you facing with? why can't the tests be migrated to JUnit 5 in this PR?

Reply:
The tests in widget.tests packages are using inherited test methods, which will work without any issue in old approach. 
However in junit 5, we need to add Test annotation for all Override test methods. I have draft those changes [here](https://github.com/RameshBabuPrudhvi/java-client/pull/1/files). However, I am just looking for any other better approach to clean up these tests. 

In this PR, we migrated tests to run in Junit5. Next PR, we can remove `vintage-engine` dependecy changes

Reply:
still have the question: why can't all the tests be migrated to JUnit Jupiter in this PR?

Reply:
Fixed Widget Tests. Please review